src/cmakeProject.ts

```diff
@@ -2452,16 +2467,19 @@ export class CMakeProject {
         }
 
         const type = targetType.label;
+        const lang = targetLang.label;
+        const langName = lang === "C++" ? "CXX" : "C";
```

Comment:
Given that [CMake's default behavior is to enable both `C` and `CXX`](https://cmake.org/cmake/help/latest/command/enable_language.html), please either change this to `C CXX` when `lang` is `C++` or add a new `C and C++` option to the quick pick.